### PR TITLE
[Slider] Fix FluentSlider two-way binding issue [#2609}

### PR DESCRIPTION
**Pull request by oneolddev**
# Pull Request

## 📖 Description

Fixes two way binding issue.

### 🎫 Issues

- setting value does not cause the slider thumbnail to redraw

## 👩‍💻 Reviewer Notes

- see comments in #2609 and #2612

### General

- [ ] I have added tests for my changes.
- [X] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [X] I have read the [CONTRIBUTING](https://github.com/microsoft/fluentui-blazor/blob/master/docs/contributing.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have added [Unit Tests](https://github.com/Microsoft/fluentui-blazor/blob/master/unit-tests.md) for my new component
- [X] I have modified an existing component
- [ ] I have validated the [Unit Tests](https://github.com/Microsoft/fluentui-blazor/blob/master/unit-tests.md) for an existing component

### CODE REVIEW

**Comment by oneolddev** [comment]
> @oneolddev please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by oneolddev** [comment]
@microsoft-github-policy-service agree

**Comment by oneolddev** [comment]
@vnbaaij 

When I attempt to rebase this over dev and run I'm getting an error - could not find `highlightElement`

image

File: eng/pipelines/build-all-lib.yml [hunk 1]
```diff
@@ -194,10 +194,10 @@ extends:
 
         # Index sources and publish symbols
 
-        - task: PublishSymbols@2
-          inputs:
-            SearchPattern: '**/bin/**/*.pdb' # string. Required. Search pattern. Default: **/bin/**/*.pdb.
-            SymbolServerType: 'TeamServices'
+        #- task: PublishSymbols@2
```
Comment (dvoituron) [inline eng/pipelines/build-all-lib.yml:197]: Why did you comment these lines?

**Review by dvoituron — DISMISSED** [review]

File: eng/pipelines/build-all-lib.yml (hunk 1 above)
Comment (vnbaaij) [reply eng/pipelines/build-all-lib.yml:197]: I did that earlier. Probably got pulled in because of rebase

**Review by vnbaaij — APPROVED** [review]

**Review by dvoituron-msft — APPROVED** [review]

**Comment by oneolddev** [comment]
Adding some links to related issues for more background - #2438 and #2591

**Pull request merged**